.travis.yml

```
@@ -56,3 +58,17 @@ jobs:
      - export PKG_CONFIG_PATH=${PKG_CONFIG_PATH}:/usr/local/lib/pkgconfig
      - ./configure --enable-libsvtav1
      - make --quiet -j$(nproc)
+
+   # GStreamer interation build
+   - name: GStreamer patch
+     script:
+     # Build and install SVT-AV1
+     - cd $TRAVIS_BUILD_DIR
+     - cd Build
+     - cmake ..
+     - make -j$(nproc)
+     - sudo make install
+     # Build GST-SVT-AV1 plugin
+     - cd $TRAVIS_BUILD_DIR/gstreamer-plugin
+     - cmake
```

```suggestion
     - cmake .
```
Make sure to add a directory